icicle-compiler/src/Icicle/Sea/FromAvalanche/State.hs

```diff
@@ -67,19 +66,19 @@ import           P
 ------------------------------------------------------------------------
 
 data SeaProgramAttribute = SeaProgramAttribute {
-    stateAttributeName  :: Int
-  , stateAttribute      :: Source.Attribute
+    stateInputIndex     :: Int
+  , stateInputId        :: InputId
```

Comment:
yeah, is a good plan, mind if I leave it for a later PR? maybe can do it when we rename SeaAttribute as we talked about